### DISCUSSION TITLE
Failed to export labels package in SLEAP 1.6.5 GUI

### DISCUSSION THREAD

**Discussion by rufeili** [talmolab/sleap#2851] [Help!]
## Description
I recently upgraded to SLEAP 1.6.5 and when I exported training job package in the GUI for remote training, an error showed up "_Failed to export labels package: ExportPackageThread.run.<locals>.on_progress() takes 2 positional arguments but 3 were given_". In comparison, export worked fine on the same .slp file when using SLEAP 1.6.1 GUI (sleap-io 0.6.4) in the past. 

It seems that sleap-io 0.9.2 changed `progress_callback` to` (current, total, phase)`, but SLEAP 1.6.5's `ExportPackageThread.run` still defines `(current, total)`. I wonder if this could be a bug and could be fixed soon?

## System 
OS: Windows 11 
SLEAP 1.6.5 · sleap-nn 0.3.3 · sleap-io 0.9.2
Python 3.13.11 · PyTorch 2.13.0+cu132 · CUDA 13.2 · RTX 4090
Install: uv tool install --python 3.13 "sleap[nn]==1.6.5" --torch-backend auto

**Comment by tom21100227** [comment]
issue #2852 , #2854 .
